Makefile

```diff
@@ -3,6 +3,9 @@ COMPANY=giantswarm
 REGISTRY=quay.io
 SHELL=bash
 MARKDOWNLINT_IMAGE=06kellyjac/markdownlint-cli:0.23.0-alpine
```

Comment:
Could we retag this?

Reply:
I made a note in the SIG Front End board